muikku-core-plugins/src/main/java/fi/muikku/plugins/websocket/WebSocketPingHandler.java

```diff
@@ -0,0 +1,16 @@
+package fi.muikku.plugins.websocket;
+
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
```

Comment:
What else? There could be no request or session active. There's no state, so no requirement for multiple instances either.